recipes/incbin/all/conanfile.py

```diff
@@ -1,32 +1,36 @@
-from conans.errors import ConanInvalidConfiguration
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.50.0"
 
-required_conan_version = ">=1.33.0"
 
 class IncbinConan(ConanFile):
     name = "incbin"
     license = "Unlicense"
     url = "https://github.com/conan-io/conan-center-index"
     description = "Include binary files in C/C++"
-    topics = ("include", "binary", "preprocess")
+    topics = ("include", "binary", "preprocess", "header-only")
     homepage = "https://github.com/graphitemaster/incbin/"
     no_copy_source = True
     settings = "compiler"
 
```

Comment:
I think you also want to specify the [package_type](https://docs.conan.io/2/reference/conanfile/attributes.html?highlight=package_type#package-type) as `header-library`.

```suggestion
package_type = "header-library"

```

Reply:
Yes, I missed that. Thank you.